www/userguide/Makefile

```diff
@@ -0,0 +1,7 @@
+all: userguide
+
+userguide:
+	doxygen ./Doxyfile
+
+clean:
+	rm -rf $(BUILD_DIR)
```

Comment:
Are you sure this variable is what you intend it to be? Is this variable even defined to a proper value? Look at how Makefile chains in `api` are laid out!